src/charts/bar.js

```diff
@@ -1122,12 +1123,17 @@ export default function module() {
      * @param  {Number} _x Desired nameLabel for the graph
      * @return {nameLabel | module} Current nameLabel or Chart module to chain calls
      * @public
+     * @deprecated
      */
     exports.nameLabel = function (_x) {
         if (!arguments.length) {
             return nameLabel;
         }
         nameLabel = _x;
+        // eslint-disable-next-line no-console
+        console.log(
```

Comment:
I think we could move this console.logs to a helper function. Its string could be taken by parameter, so we could:

- Remove `eslint-disable` from several lines and just have it on the helper.
- Make the code more dry.

Also, wouldn't it be more appropriate to use `console.warn` or `console.info` instead of log ? What do you think @Golodhros ?

Reply:
Yeah, after the first three I was thinking the same!

Good call, console.warn will fit better!

Reply:
:+1: :+1: :+1: